browser/resources/settings/default_brave_shields_page/default_brave_shields_page.html

```diff
@@ -35,12 +35,7 @@
       <div class="label shields-primary-title">$i18n{braveShieldsDefaultsSectionTitle}</div>
       <div class="label secondary shields-secondary-title">
         <p>$i18n{braveShieldsDefaultsSectionDescription1}</p>
-        <template is="dom-if" if="[[isBraveRewardsSupported_]]">
```

Comment:
Looks like `isBraveRewardsSupported_` can be removed from the corresponding .ts file